src/App.css

```diff
@@ -0,0 +1,253 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto+Slab:wght@300&display=swap');
+
+:root {
+  --main-acent-color: #0290ff;
+}
+
+body {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  font-family: Montserrat, 'Times New Roman', Times, serif;
+  background-color: whitesmoke;
+}
+
+.d-flex {
+  display: flex;
+}
+
+.d-flex h1 {
+  font-size: 30px;
+  font-family: Montserrat, serif;
+  font-weight: bold;
+}
+
+.d-flex li {
+  font-size: 14px;
+  font-family: Montserrat, serif;
+}
+
+/* .navigation_container {
+  background: #fff;
+  border-bottom: 1px solid #e8e8e8;
+} */
+
```

Comment:
- Please remove all commented code blocks from your code. This would help keep your code clean and make it easy to maintain 👍 👍

Reply:
Remove commented code